listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -454,12 +581,21 @@ def get_recording_pin_events(users_for_events: List[dict], min_ts: int, max_ts:
                     )
                 )
             )
+            
+            hidden = False
```

Comment:
Same  as above.

Reply:
Actually also no need to 2 queries for this, my suggestion is do the database query for all events types at once and then in python build 2 dicts of different events and do the hiding.